### DISCUSSION TITLE
Can mitmproxy reverse mode map a service to a subdir?

### DISCUSSION THREAD

**Discussion by proofrock** [mitmproxy/mitmproxy#5326] [Q&A] [answered]
Hi,
   I need to use reverse proxy mode (I think) to remap a service of mine (let's say `http://localhost:17178`) to a subpath of another address (e.g. `http://localhost:8080/mypath/`) so that all the requests to that subpath get sent to the service. It's for debugging, not for production.

Is this possible, and could you point me to the right documentation? I think I don't know the right terms to look for in the documentation.

Thanks, and best regards,

       Germano

**Comment by mhils** [comment, marked as answer]
I think the easiest way to do this is to use the [Map Remote](https://docs.mitmproxy.org/stable/overview-features/#map-remote) feature. :)

**Comment by proofrock** [comment]
Thanks, I will try! Thanks for a wonderful product.
